pkg/scaffold/v2/controller/controller_suitetest.go

```diff
@@ -124,24 +124,24 @@ var _ = AfterSuite(func() {
 
 // Update updates given file (suite_test.go) with code fragments required for
 // adding import paths and code setup for new types.
-func (a *ControllerSuiteTest) Update() error {
+func (v *SuiteTest) Update() error {
```

Comment:
@Adirio IIRC you have already changed the receiver name, right?

Reply:
Yes, all them have been changed.

Reply:
I'd recommend we merge this PR first and then use the lint to verify the changes made in the receiver in the @Adirio's PR automatically via the lint.

Reply:
They do pass the lint check. I don't really think the order matters that much, the one who gets merged latter will need a rebase, in my case to change again to `f` in your case to remove this change.

Reply:
@Adirio  the golint check is not enabled in the master(it is the motivation of this PR)? Did you enable it locally to verify the changes made in your PR?

Reply:
I know, I used my local lint. My PR solves using different pointers for the same structs (which is a part of golint, yours handle the whole golint checks).

Reply:
It is fine. If your PR be merged before your change will be in the rebase with. If this is merged before, then the same can be done when your PR be rebased with the master. One PR should not bock the other anyway. 

c/c @mengqiy